modules/rup/routes/prestacion.ts

```diff
@@ -311,6 +311,10 @@ router.get('/prestaciones/solicitudes', (req, res, next) => {
         query.where('paciente.id').equals(req.query.idPaciente);
     }
 
+    if (req.query.origenTOP) {
```

Comment:
Yo haría el origen un string, y ya agregaría el caso de que sea igual a RUP, es decir que tenga prestación origen. Así ya la API queda preparada para ambos filtros. 
 
```suggestion
    if (req.query.origen === 'top') {
```